packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -56,6 +61,7 @@ const PdfViewer: FC<Props> = ({
   scale,
   document,
   textLayerClassName,
+  disableTextLayer,
```

Comment:
I know that `undefined` is falsy, but it might make sense to give this a default value of `false`.